app/models/validators/aaib_report_validator.rb

```diff
@@ -9,5 +9,5 @@ class AaibReportValidator < SimpleDelegator
   validates :summary, presence: true
   validates :body, presence: true, safe_html: true
 
-  validates :date_of_occurrence, presence: true, date: true
+  validates :date_of_occurrence, date: true, allow_blank: true
```

Comment:
Going by the A/C: `AAIB can leave the 'date of occurrence' blank for this new report type only`. Won't this change allow blank dates across the board/